style.css

```diff
@@ -0,0 +1,196 @@
+*
+{
+    margin: 0;
+    padding: 0;
+    font-family: poppins;
```

Comment:
I think you can use font-family: "poppins", san serif; to create a sense of formality and elegance.

Reply:
Kindly use some emoji and GIF smile to make your comments more friendly :hugs: